structuredefinitions/Extension-UKCore-SubscriptionHeaderVersion.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-UKCore-SubscriptionHeaderVersion" />
+  <url value="http://hl7.org/fhir/5.0/StructureDefinition/extension-Subscription.parameter" />
```

Comment:
Should Extension be capitalised in the URL?

Reply:
Based on the guidance [here](https://build.fhir.org/versions.html#extensions). The format is lower case.